lib/srv/regular/sshserver_test.go

```diff
@@ -1453,14 +1496,16 @@ func TestLimiter(t *testing.T) {
 	require.NoError(t, auth.CreateUploaderDir(nodeStateDir))
 	defer srv.Close()
 
-	// maxConnection = 3
-	// current connections = 1 (one connection is opened from SetUpTest)
 	config := &ssh.ClientConfig{
 		User:            f.user,
 		Auth:            []ssh.AuthMethod{ssh.PublicKeys(f.up.certSigner)},
 		HostKeyCallback: ssh.FixedHostKey(f.signer.PublicKey()),
 	}
 
+	require.Eventually(t, testHelperGetWaitForServerStart(t, srv, config), time.Second*10, time.Millisecond*100)
```

Comment:
Is this needed? Wouldn't the dial that this method does interfere with the dials that test does itself?

Reply:
You are right this isn't needed. I'll remove it now. 
Would it interfere with the rest since the connection is closed?